docs/ref/clvm.md

```diff
@@ -603,53 +603,224 @@ When used in an integer context, nil behaves as zero.
-| `>` | 498 | - | 2 |
-| `>s` | 117 | - | 1 |
-| `strlen` | 173 | - | 1 |
-| `concat` | 142 | 135 | 3 |
-| `divmod` | 1116 | - | 6 |
-| `sha256` | 87 | 134 | 2 |
-| `ash` | 596 | - | 3 |
-| `lsh` | 277 | - | 3 |
-| `not`, `any`, `all` | 200 | 300 | - |
-| `point_add` | 101094 | 1343980 | - |
-| `pubkey_for_exp` | 1325730 | - | 38 |
-| | | | |
-| `CREATE_COIN` | 1800000 | - | - |
-| `AGG_SIG_UNSAFE`,`AGG_SIG_ME` | 1200000 | - | - |
-
-Aside from cost, the maximum number of atoms or pairs in a CLVM program is 2^31. If this threshold is exceeded, the program will be terminated and no value will be returned.
\ No newline at end of file
+| operator            | base cost | cost per arg | cost per byte |
+| ------------------- | --------- | ------------ | ------------- |
+|                     |           |              |               |
+| `f` *first*         | 30        | -            | -             |
+| `i` *if*            | 33        | -            | -             |
+| `c` *cons*          | 50        | -            | -             |
+| `r` *rest*          | 30        | -            | -             |
+| `l` *listp*         | 19        | -            | -             |
+| `q` *quote*         | 20        | -            | -             |
+| `a` *apply*         | 90        | -            | -             |
+| `=`                 | 117       | -            | 1             |
+| `+`                 | 99        | 320          | 3             |
+| `/`                 | 988       | -            | 4             |
+| `*`                 | 92        | 885          | [see here](https://github.com/Chia-Network/clvm_tools/blob/main/costs/README.md#multiplication) |
+| `logand`, `logior`, `logxor` | 100 | 264       | 3             |
+| `lognot`            | 331       | -            | 3             |
+| `>`                 | 498       | -            | 2             |
+| `>s`                | 117       | -            | 1             |
+| `strlen`            | 173       | -            | 1             |
+| `concat`            | 142       | 135          | 3             |
+| `divmod`            | 1116      | -            | 6             |
+| `sha256`            | 87        | 134          | 2             |
+| `ash`               | 596       | -            | 3             |
+| `lsh`               | 277       | -            | 3             |
+| `not`, `any`, `all` | 200       | 300          | -             |
+| `point_add`         | 101094    | 1343980      | -             |
+| `pubkey_for_exp`    | 1325730   | -            | 38            |
+|                     |           |              |               |
+| `CREATE_COIN`       | 1800000   | -            | -             |
+| `AGG_SIG_UNSAFE`,`AGG_SIG_ME` | 1200000 | -    | -             |
+
+Aside from cost, the maximum number of atoms or pairs in a CLVM program is 2^31. If this threshold is exceeded, the program will be terminated and no value will be returned.
+
+## Evaluating cost for a sample brun program
```

Comment:
I like this section, it's good to describe this in detail. However, I think we should hold off on publishing this until `brun` behaves well and returns the correct cost. Then this section could be simplified and more logical.